crates/cast/tests/cli/main.rs

```diff
@@ -122,6 +122,17 @@ Created new encrypted keystore file: [..]
     ]);
 });
 
+// tests that we can create a new wallet with keystore
+casttest!(new_wallet_default_keystore_with_password, |_prj, cmd| {
+    cmd.args(["wallet", "new", "--unsafe-password", "test"]).assert_success().stdout_eq(str![[
+        r#"
+Created new encrypted keystore file: [..]
+[ADDRESS]
+
+"#
+    ]]);
+});
+
```

Comment:
Can you add an assertion that checks for the existence / creation of the default keystore location and an additional test that checks there is no regression in regards to outputting multiple keys without a keystore?